dist/save/index.js

```diff
@@ -41063,33 +41063,34 @@ const utils = __importStar(__webpack_require__(443));
 process.on("uncaughtException", e => utils.logWarning(e.message));
 function saveImpl(stateProvider) {
     return __awaiter(this, void 0, void 0, function* () {
+        let cacheId;
         try {
             if (!utils.isCacheFeatureAvailable()) {
-                return;
+                return 0;
```

Comment:
We will return 0 when we don't want to log a warning, and -1 when we want to log a warning.